core/src/main/java/nl/nn/adapterframework/configuration/ConfigurationDigester.java

```diff
@@ -100,15 +100,15 @@
 	private class XmlErrorHandler implements ErrorHandler  {
 		@Override
 		public void warning(SAXParseException exception) throws SAXParseException {
-			ConfigurationWarnings.add(log, "Warning when validating against schema ["+CONFIGURATION_VALIDATION_SCHEMA+"] at line,column ["+exception.getLineNumber()+","+exception.getColumnNumber()+"]: " + exception.getMessage());
+			ConfigurationWarnings.addGlobalWarning(log, "Warning when validating against schema ["+CONFIGURATION_VALIDATION_SCHEMA+"] at line,column ["+exception.getLineNumber()+","+exception.getColumnNumber()+"]: " + exception.getMessage());
```

Comment:
These warnings are very configuration specific. If you pass the Configuration that is parsed in the constructor of XmlErrorHandler, than you can make these warnings configuration specific